lib/logger.js

```diff
@@ -141,13 +141,20 @@ function Logger(key, options) {
     };
 
     const useHttps = configs.AGENT_PROTOCOL === 'https';
+    let headers;
+
+    if (typeof window === 'object') {
+        headers = Object.assign(configs.DEFAULT_REQUEST_HEADER, {'source': `browser/${pkg.version}`});
```

Comment:
what about changing this to
```
`${options.UserAgent || configs.DEFAULT_USER_AGENT} (browser)`
```
?

Because in the formula it is `<pkg.name>/<pkg.version> (<system info>)` and we don't have a package named `browser` and if we keep it as my suggestion, we won't need to update the backend, as I see.

Reply:
```
Object.assign(configs.DEFAULT_REQUEST_HEADER, {'source': `browser/${pkg.version}`})
```
will change the global config variable `DEFAULT_REQUEST_HEADER` which is not expected. A better way and more performant way to do it is
```
headers = {
    ...configs.DEFAULT_REQUEST_HEADER
    , source: `browser/${pkg.version}`
}
```

Reply:
I don't quite understand what you mean, @smusali.

Reply:
What I meant there was why you decided to use 'browser/<version>' instead of 'logdna/<version> (browser)'? Here the package name is logdna and we agreed having user-agent field to start with '<pkg.name>/<pkg.version>', right? Your current decision, which is 'browser/<version>', violates this agreement and consistency

Reply:
@smusali  the aggregated data structure looks like that: 
`{logdna: {1.1.1: 1. 1.1.2: 2, 5.5.5: 9}` @andkon wants to have information for how many people are running node library for their front-end projects (in browsers). So I am recording the package name as a 'browser` here. Ingester  performs its sanitation with strict rules for the forma like this (logdna/1.2.3) where the pkg name must be on of the listed package names and the string after the `/` has to have this format `1.2.3`.

Reply:
@vilyapilya, just checked internally by running unit testing using `logdna/1.2.3 (browser)` in `user-agent` header - still works and passes a sanitization test; so, if that's working well, I prefer keeping `<pkg.name>/<pkg.version> (<sysInfo>)` format and not changing `pkg.name` from `logdna` to `browser`